gst/nnstreamer/tensor_typedef.h

```diff
@@ -78,6 +78,32 @@
      * "dimensions = (string) dim1:dim2:dim3:dim4, dim1:dim2:dim3:dim4"
      */
 
+#define GST_TENSOR_VIDEO_CAPS_STR \
+    GST_VIDEO_CAPS_MAKE ("{ RGB, BGR, RGBx, BGRx, xRGB, xBGR, RGBA, BGRA, ARGB, ABGR, GRAY8 }") \
```

Comment:
1. please reconsider to move media-caps definition. (GST_TENSOR_MEDIA_CAPS_STR)
GST_VIDEO_CAPS_MAKE / GST_AUDIO_CAPS_MAKE macros are defined in gst/video/video-format.h gst/audio/audio-format.h
IMO, it is better to define these in _plugin_api.h

2. About nnstreamer_plugin_api.c, do we need to separate util functions into nnstreamer_plugin_api.c?

Reply:
>     1. ...     it is better to define these in _plugin_api.h

If they are only used in subplugins, yes. I'll look into it.

>     2\. About nnstreamer_plugin_api.c, do we need to separate util functions into nnstreamer_plugin_api.c?

We've just lost the need for this.

Reply:
> We've just lost the need for the separation. We can move back them to common.c.

:sob:
If we are going to melt nnstreamer_plugin_api.c into common.c, I want to let that .c file go by my own hands. :smiley: